ps_smr.h

```diff
@@ -28,40 +28,17 @@
 #include <ps_config.h>
 #include <ps_global.h>
 #include <ps_slab.h>
-
-#ifndef PS_QLIST_BATCH
-#define PS_QLIST_BATCH 128
-#endif
-
-struct ps_quiescence_timing {
-	volatile ps_tsc_t     time_in, time_out;
-	volatile ps_tsc_t     last_known_quiescence;
-	char __padding[PS_CACHE_PAD_SZ(3*sizeof(ps_tsc_t))];
-} PS_ALIGNED PS_PACKED;
-
-struct __ps_other_core {
-	ps_tsc_t time_in, time_out, time_updated;
-};
-
-struct ps_smr_percore {
-	/* ps_quiescence_timing info of this CPU */
-	struct ps_quiescence_timing timing;
-	/* ps_quiescence_timing info of other CPUs known by this CPU */
-	struct __ps_other_core timing_others[PS_NUMCORES];
-	/* padding an additional cacheline for prefetching */
-	char __padding[PS_CACHE_PAD_SZ(sizeof(struct __ps_other_core)*PS_NUMCORES + sizeof(struct ps_quiescence_timing))];
-} PS_ALIGNED PS_PACKED;
+#include <ps_quiesce_type.h>
```

Comment:
Not sure if the file is best described as the "type".  Perhaps just `ps_quiescence.h`?